File: Python/11522.py
class Calc:
	def __init__(self, n):
		self.until = n
		self.sum1 = 0

		self.Answer()
		
	def Answer(self):

		self.sum1 = sum(range(1, self.until + 1))
	
	def Show(self):
		print("%d %d %d" % (self.sum1, self.sum1 * 2 - self.until, self.sum1 * 2))

def main():
	test_case = int(input())
	for i in range(test_case):
		set_number, N = map(int, input().split(' '))
		data = Calc(N)
		print(set_number, end = ' ')
		data.Show()

if __name__ == "__main__":
	main()
		
		

File: C++/11522.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

class Calc
{
private:
    int sum_1;
    int until;
public:
    Calc()
    {
        sum_1 = 0;
    }

    void Setup(int x)
    {
        until = x;
    }
    void Answer()
    {
        for (int i = 1; i <= until; i++){
            sum_1 += i;
        }
    }
    void Show() const
    {
        cout<<sum_1<<" "<<sum_1 * 2 - until<<" "<<sum_1 * 2<<endl;
    }

};

int main()
{
    int test_case;
    int N;
    int set_number;
    Calc data;
    cin>>test_case;
    for (int i = 1; i <= test_case; i++){
        Calc data;
        cin>>set_number;
        cin>>N;
        data.Setup(N);
        data.Answer();
        cout<<set_number<<" ";
        data.Show();

    }
    return 0;
}










